parsingTest/intermediate/parserImpls.ftl

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Would it be better to move these files into something like a "test/resources" folder so there doesn't need to be a new folder on the root directory?

Reply:
Do you mean where the test files for processFile are?

Reply:
Yeah, maybe a new folder in the same test folder where processFileTests is

Reply:
I like that idea, I'll try it out!